src/test/kotlin/org/opensearch/commons/utils/TestHelpers.kt

```diff
@@ -16,7 +17,11 @@ import java.io.ByteArrayOutputStream
 fun getJsonString(xContent: ToXContent): String {
     ByteArrayOutputStream().use { byteArrayOutputStream ->
         val builder = XContentFactory.jsonBuilder(byteArrayOutputStream)
-        xContent.toXContent(builder, ToXContent.EMPTY_PARAMS)
+        if (xContent is CorrelationAlert) {
```

Comment:
do we need this change? because this is not part of alerting package.

Reply:
yeah I'm using `getJsonString` method to convert correlation alert Json into string on line 87 in src/test/kotlin/org/opensearch/commons/alerting/CorrelationAlertTests.kt therefore changed this to make it work for CorrelationAlert